CMakeLists.txt

```diff
@@ -2,8 +2,9 @@ cmake_minimum_required(VERSION 3.15)
 
 if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
   set(STRICT_COMPILE_FLAGS -Werror=all
+                           -Werror=float-conversion
                            -Werror=inconsistent-missing-override
-                           -Werror=float-conversion)
+                           -Werror=unused-parameter)
```

Comment:
I think GCC supports unused-parameter, should we add it there as well?

Reply:
I think it is enough if we enable warning enforcement in clang.  I personally think we should default Orbit on clang and drop gcc support. There seems to be no advantage in having support for it.

Reply:
GCC is the default Ubuntu compiler.  That's the one I use on my Ubuntu box, it's not that much work to keep supporting it...  Don't worry for this PR, I'll test it on my side and update the GCC flags.